hack/install.sh

```diff
@@ -120,6 +120,10 @@ do_install() {
 	if command_exists lsb_release; then
 		lsb_dist="$(lsb_release -si)"
 		dist_version="$(lsb_release --codename | cut -f2)"
+		if [ "$lsb_dist" = "elementary OS" ] && [ "$dist_version" = "freya" ]; then
```

Comment:
are there other versions, i feel like this will eventually become a switch with multiple